yt/utilities/tests/test_input_validator.py

```diff
@@ -0,0 +1,15 @@
+from nose.tools import assert_raises, assert_equal
+
+from yt.testing import fake_random_ds
+from yt.utilities.exceptions import YTInvalidArgumentType
+
+
+def test_bad_disk_input():
```

Comment:
This also seems like a test that belongs in `yt/data_objects` with the thing it's testing.

Reply:
moved